modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/15_info_ingest.yml

```diff
@@ -65,6 +65,8 @@ teardown:
   - gte: { ingest.pipelines.ingest_info_pipeline.time_in_millis: 0 }
   - match: { ingest.pipelines.ingest_info_pipeline.current: 0 }
   - match: { ingest.pipelines.ingest_info_pipeline.failed: 0 }
+  - gt: { ingest.pipelines.ingest_info_pipeline.ingested_in_bytes: 0 }
+  - gt: { ingest.pipelines.ingest_info_pipeline.produced_in_bytes: 0 }
```

Comment:
I'm adding a comment here because I can't open one at an unchanged line.

I learned that I gave you some incorrect info about pipeline cleanup. Apparently, indices (and some other stuff) are cleanup up automatically at the end of a yaml file, but _not_ between tests within a yaml file. I learned this from reusing an index in two tests and getting an unexpected number of counts.

On the other hand, pipelines are _not_ cleaned up automatically at the end of the file. So we should add back the code you had to delete all the pipelines. Also, a pipeline cannot be deleted if it is a default or final pipeline for an index, so you probably have to explicitly delete such indices in the teardown block before deleting the pipelines.
Interestingly, the teardown block is called after every test within a yaml file. So if you delete an index in the teardown block, you don't have to make sure to use a different index name in different  tests within the same yaml file.

Here's some docs with some of this info: https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc